tests/python/unittest/test_runtime_profiling.py

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import tvm
+import time
+
+
+@tvm.testing.parametrize_targets
+def test_cpu_timer(target, ctx):
+    timer_stop = tvm.runtime.start_timer(ctx)
+    time.sleep(1e-3)
+    nanosecs = timer_stop()
+    assert nanosecs < 2e6 and nanosecs > 5e5
```

Comment:
I don't think we should do this, it's inviting a flaky test. do we have a mock DLContext?

Reply:
I've switch to a test that just checks that the time is > 0.